packages/reporters/dev-server/src/Server.js

```diff
@@ -40,6 +40,9 @@ function setHeaders(res: Response) {
     'Access-Control-Allow-Headers',
     'Origin, X-Requested-With, Content-Type, Accept, Content-Type',
   );
+  res.setHeader('Cross-Origin-Opener-Policy', 'same-origin');
+  res.setHeader('Cross-Origin-Resource-Policy', 'same-origin');
+  res.setHeader('Cross-Origin-Embedded-Policy', 'require-corp');
```

Comment:
```suggestion
  res.setHeader('Cross-Origin-Opener-Policy', 'same-origin');
  res.setHeader('Cross-Origin-Embedder-Policy', 'require-corp');
```

1. It's called `Cross-Origin-Embedder-Policy `: https://developer.chrome.com/blog/enabling-shared-array-buffer/#cross-origin-isolation
2. As far as I can tell, `Cross-Origin-Resource-Policy` isn't really needed here

Reply:
Fixed the typo,  good catch again,  I'm sorry for causing many iterations from you side for a task that should've been a quick win :|